src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
```

Comment:
I did this a little differently, and used 2 * `space-md` to get the space I needed. Hopefully by using a multiple of one of our standard sizes, I'm being more modular.